packages/commonwealth/client/scripts/controllers/server/sessions.ts

```diff
@@ -0,0 +1,24 @@
+import { ethers } from 'ethers';
+
+class SessionsController {
+  wallets: Record<string, ethers.Wallet>;
```

Comment:
What's the use case / point of this controller? Only for eth?

Reply:
@rjwebb's adding other chains to this controller in PR4 - it should support every cryptography scheme that we cover.